src/MCPClient/lib/clientScripts/archivematicaTranscribeFile.py

```diff
@@ -8,18 +8,14 @@
 from tempfile import gettempdir
 from uuid import uuid4
 
-path = '/usr/share/archivematica/dashboard'
-if path not in sys.path:
-    sys.path.append(path)
+import django
+sys.path.append('/usr/share/archivematica/dashboard')
```

Comment:
I wonder if it's worth changing MCPClient to export `PYTHONPATH` and `DJANGO_SETTINGS_MODULE` when spinning up subprocesses? That way we could avoid requiring every client script to set its own `sys.path`.